bin/unpublish-tags.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+dockerhub_token=$(curl -s -f -H "Content-Type: application/json" -X POST -d "{\"username\": \"${DOCKER_HUB_USERNAME}\", \"password\": \"${DOCKER_HUB_TOKEN}\"}" https://hub.docker.com/v2/users/login/ | jq -r .token)
+
+unpublish_group() {
+    local stackVersion="$1"
+    local targetTagSuffix="$2"
+    local status=0
+    variants=("" "-build")
+    if (( stackVersion <= 22 )); then
+        variants+=("-cnb" "-cnb-build")
+    fi
+    for variant in "${variants[@]}"; do
+      echo "Deleting heroku/heroku:${stackVersion}${variant}${targetTagSuffix}"
+        response=$(curl -s -X DELETE \
+            -H "Authorization: JWT ${dockerhub_token}" \
+            "https://hub.docker.com/v2/namespaces/heroku/repositories/heroku/tags/${stackVersion}${variant}${targetTagSuffix}"
```

Comment:
The constructed tag name (`${stackVersion}${variant}${targetTagSuffix}`) is repeated twice - once in the log output and once in the URL. Should it be assigned to an intermediate variable to avoid the two getting out of sync in the future?

Reply:
Done in https://github.com/heroku/base-images/pull/245/commits/e582ef7f2e9fc32f9bf6d6dc9b83500f41acc3d2.